source/_integrations/unifiled.markdown

```diff
@@ -0,0 +1,56 @@
+---
+title: "UniFi LED"
+description: "Instructions on how to configure the UniFi LED integration with UniFi LED Controller by Ubiquiti."
+logo: ubiquiti.png
+ha_category:
+  - Light
+ha_release: next
+ha_iot_class: Local Polling
+---
+
+[UniFi LED](https://unifi-led.ui.com/) by [Ubiquiti Networks, inc.](https://www.ubnt.com/) is a system off controler managed led light panels and dimmers.
+
+There is currently support for the following device type within Home Assistant:
+
+- [Light](#light)
+
+## Configuration
+
+This integration uses settings in the configuration file. Underneath here is a example.
+
+```yaml
+# Example configuration.yaml entry
+light:
+  - platform: unifiled
+    host: 192.168.1.20
+    port: 20443
+    username: ubnt
+    password: ubnt
+```
+
+{% configuration %}
```

Comment:
Is there also the option ID? [See here](https://github.com/home-assistant/home-assistant/pull/27475/files#diff-6ff9207b02057a414a71a1e032da1acaR31)

Reply:
That is something left over from testing, removing it from the integration.